desk.acc/this.apple.s

```diff
@@ -1064,36 +1064,49 @@ notpas:
 ;;; ============================================================
 ;;; Calculate RamWorks memory; returns number of banks in Y
 ;;; (256 banks = 0)
-;;; Inspired by "gid" comp.sys.apple2.programmer
+;;; non-destructive version
+;;; note the bus floats for RamWorks RAM when the bank has no RAM.
```

Comment:
Oh, awesome. It was unclear to me from the code I looked at if the behavior was that reads would be floating or if the bank switch would just fail.